bodhi/client/bindings.py

```diff
@@ -425,6 +426,10 @@ def list_overrides(self, user=None, packages=None,
             releases (basestring): Comma separated Release shortnames to filter buildroot overrides
                 by.
             builds (basestring): Comma separated build NVRs to filter overrides by.
+            rows_per_page (int): Limit the results to a certain number of rows per page.
+                (default:20)
+            page (int): Return a specific page of results.
+                (default:1)
```

Comment:
I would not document these defaults here, because they are API defaults and not bindings defaults. This means that if the API defaults ever change it is unlikely that we will remember to change this here and it'll then be incorrect. Instead, I would just document that the value None is the default, which will cause it to use the API default.